ea_restaurant_data_loader/test/lib/service/app_client_service_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule EaRestaurantDataLoader.AppClientServiceTest do
+  use ExUnit.Case, async: true
+
+  alias EaRestaurantDataLoader.AppClientService
+  alias EaRestaurantDataLoader.AppClient
+
+  import Mock
+
+  describe "app client service test" do
+    test "get app client by client id should return expected app client" do
```

Comment:
The test is wrong, you should be mocking `Repo` from ecto and `AppClientService`

Reply:
Done.